projects/atft/src/lib/controls/orbit-controls.directive.ts

```diff
@@ -83,18 +84,21 @@ export class OrbitControlsDirective implements AfterViewInit, OnChanges, OnDestr
     );
     this.controls.rotateSpeed = this.rotateSpeed;
     this.controls.zoomSpeed = this.zoomSpeed;
-    this.controls.addEventListener('change', this.childRenderers.first.render);
-    this.childRenderers.first.render();
+
+    this.requestRender = this.requestRender.bind(this);
+    this.controls.addEventListener('change', this.requestRender);
```

Comment:
Usually, I'd say this look dubious since `this` will be lost in `requestRender` or at least not what you would expect.